AwsCryptographyPrimitives/runtimes/go/ImplementationFromDafny-go/ExternDigest/externs.go

```diff
@@ -0,0 +1,30 @@
+package ExternDigest
+
+import (
+	"crypto"
+	"fmt"
+	"hash"
+
+	"github.com/aws/aws-cryptographic-material-providers-library/primitives/AwsCryptographyPrimitivesTypes"
+	"github.com/dafny-lang/DafnyRuntimeGo/v4/dafny"
+	"github.com/dafny-lang/DafnyStandardLibGo/Wrappers"
+)
+
+func Digest(algorithm AwsCryptographyPrimitivesTypes.DigestAlgorithm, message dafny.Sequence) Wrappers.Result {
+	hash, _ := getNativeEcdhCurve(algorithm)
+	hash.Write(dafny.ToByteArray(message))
+	return Wrappers.Companion_Result_.Create_Success_(dafny.SeqOfBytes(hash.Sum(nil)))
+}
+
+func getNativeEcdhCurve(algorithm AwsCryptographyPrimitivesTypes.DigestAlgorithm) (hash.Hash, error) {
```

Comment:
Let's rename this, its name is inaccurate.
What about getNativeDigestAlgorithm?